style.css

```diff
@@ -73,21 +73,289 @@
+  font-size: 13px;
+  line-height: 16px;
+  display: flex;
+  justify-content: space-between;
+  width: 80%;
+  margin: 15px 0;
+}
+
+.client {
+  color: #344563;
+}
+
+.dim {
+  color: #7a869a;
+}
+
+.project-desc {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin: 10px 10px 10px 0;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 8px;
+}
+
+.tag {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 4px 12px;
+}
+
+.btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  width: 124px;
+  height: 48px;
+  margin: 15px 0;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  cursor: pointer;
+}
+
+.btn:hover {
+  background-color: #396df2;
+  color: white;
+}
+
+.about-myself {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  margin-top: 100px;
+  background-color: #fff;
+  border-top-right-radius: 73px;
+}
+
+.top {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.about-title {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+}
+
+.about-content {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  padding-top: 15px;
+  color: #344563;
+}
+
+.about-sm {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 16px;
+}
+
+.supporting-text {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 30px;
+  color: #7f8cff;
+}
+
+.about-smedia-icons {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 15px;
+}
+
+.btn2 {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 12px;
+  gap: 10px;
+  margin: 15px 0;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  background-color: #fff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  cursor: pointer;
+}
+
+.btn2:hover {
+  background-color: #396df2;
+  color: white;
+}
+
+.experties {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 0;
+  order: 1;
+  width: 100%;
+}
```

Comment:
- [ ] **Good job on replicating the Figma design 👍** However, it would be better also to replicate the alignment and margin of your contents as given on the Figma template, I have highlighted some to be fixed 😎😎

**Your design** | **Figma design**
| --- | --- |
![image](https://user-images.githubusercontent.com/102757126/219710541-393d5c2f-ecf9-4cc3-8d45-9e191ee02864.png) | ![image](https://user-images.githubusercontent.com/102757126/219710827-c195e143-1167-48f7-a547-1c4f7e572183.png)


### KEEP IT UP 👍